pkg/controller/operators/olm/operator.go

```diff
@@ -32,19 +36,24 @@ import (
 var ErrRequirementsNotMet = errors.New("requirements were not met")
 var ErrCRDOwnerConflict = errors.New("CRD owned by another ClusterServiceVersion")
 
+//TODO(jpeeler): copied from catalog/operator.go
+var timeNow = func() metav1.Time { return metav1.NewTime(time.Now().UTC()) }
+
 const (
 	FallbackWakeupInterval = 30 * time.Second
 )
 
 type Operator struct {
 	*queueinformer.Operator
-	csvQueue    workqueue.RateLimitingInterface
-	client      versioned.Interface
-	resolver    install.StrategyResolverInterface
-	lister      operatorlister.OperatorLister
-	annotator   *annotator.Annotator
-	recorder    record.EventRecorder
-	cleanupFunc func()
+	csvQueue            workqueue.RateLimitingInterface
+	client              versioned.Interface
+	resolver            install.StrategyResolverInterface
+	lister              operatorlister.OperatorLister
+	csvLister           map[string]csvlister.ClusterServiceVersionLister
```

Comment:
After some fixes in [PR-536](https://github.com/operator-framework/operator-lifecycle-manager/pull/536) get in, we can update this to extend `operatorlister.OperatorLister`.